### PR TITLE
Merge navbar into main

### PR DESCRIPTION
**Pull request by tianwanlin**
### Description
This pull request introduces a new design for the navbar component.

### Type of change
- [ ] Design Change

### Demo
image

### CODE REVIEW

**Review by lionelgwk — APPROVED** [review]
lgtm!

File: my-app/src/pages/components/navbar.tsx [hunk 1]
```diff
@@ -53,44 +53,38 @@ const Sidebar = () => {
...
-              <div className="opacity-60 cursor-pointer">
+              <div className="opacity-50 hover:opacity-100 hover:text-pri-500">
                 <IoNotificationsOutline size={25}/>
               </div>
             </Badge>
           </div>    
         </PopoverTrigger>
         <PopoverContent>
           <div className="px-1 py-2 h-fit w-48">
-            <div className="font-bold">Notifications</div>
```
Comment (jteoo) [inline my-app/src/pages/components/navbar.tsx]: tetx or text?

Comment (jteoo) [reply my-app/src/pages/components/navbar.tsx]: if dont need please remove

**Review by jteoo — COMMENTED** [review]
please review comments and commit again

**Review by jteoo — APPROVED** [review]

**Pull request merged**